packages/hash/api/src/graphql/resolvers/knowledge/comment/update.ts

```diff
@@ -0,0 +1,48 @@
+import { WORKSPACE_TYPES } from "../../../../graph/workspace-types";
```

Comment:
This mutation isn't doing anything comment-specific at the moment. If it stays how it is, you probably don't need it and can just use `updateEntity`

I suggest something useful it could do is check if a user is trying to update the `deletedAt` property and then the server overwrite it, to stop people faking deletion time. The same could be done for `resolvedAt`

Reply:
Replaced the mutation with `deletePersistedComment` in e54545814ea2ab44fa446fadc368585a1f40cd9f to prevent deleted timestamps from being faked. Will take a similar approach for resolving comments. Thank you for the suggestion.